src/components/Loader/Loader.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import './Loader.css';
+
+
+export type LoaderProps = {
+    /** Размер */
+    size?: 's' | 'm' | 'l';
+    /** Дополнительный класс */
+    className?: string;
+    fill?: string;
+};
+
+const Loader: React.FC<LoaderProps> =({size,className}) => (
```

Comment:
[!] получается, по умолчанию, size === l, тогда ниже можно убрать проверку на size === undefined
```suggestion
const Loader: React.FC<LoaderProps> =({ size = l, className }) => (
```